jax/_src/scipy/sparse/linalg.py

```diff
@@ -277,15 +274,37 @@ def cg(A, b, x0=None, *, tol=1e-5, atol=0.0, maxiter=None, M=None):
       inverse of A.  Effective preconditioning dramatically improves the
       rate of convergence, which implies that fewer iterations are needed
       to reach a given error tolerance.
+  symmetric : bool, optional
+      Whether the linear operator is symmetric. If true then it is assumed
+      A @ b == A.T @ b, i.e. matvec == vecmat. When this is true, reverse
+      mode differentiation is cheaper. If not specified, JAX will attempt to
+      determine this automatically. Note, if the linear operator is
+      real-valued, it is symmetric, because it's already positive-definite.
+      Additionally, if the linear operator is complex-valued, but Imag(A) = 0,
+      it is symmetric, however automatic detection would fail.
```

Comment:
Simpler to say that it defaults to `True` for real-valued systems, and `False` for complex-valued systems.